pkg/model/test.go

```diff
@@ -19,6 +19,7 @@ type Test struct {
 	Context   string        `yaml:"context,omitempty"`
 	Commands  []TestCommand `yaml:"commands,omitempty"`
 	DependsOn []string      `yaml:"depends_on,omitempty"`
+	Cache     []string      `yaml:"cache,omitempty"`
```

Comment:
I don't see a custom unmarshall in order to have the following syntax:

```yaml
tests:
  cache: my-image
```

We already have that in the compose spec. Should we be doing it here?

Reply:
Should we allow this? cc @pchico83

Reply:
For simplicity, I would go with a list of strings for now.
But I would rename the field to `caches`